src/axom/mint/mesh/UnstructuredMesh.hpp

```diff
@@ -140,11 +205,6 @@ class UnstructuredMesh : public Mesh
    */
   using CellToNodeConnectivity =
     ConnectivityArray<topology_traits<TOPO>::cell_to_nodes>;
```

Comment:
@kanye-quest , it looks like you replaced all the `ConnectivityArray` relations with Slam types, except for this one.  Why was that?

Reply:
Expanding: Why have  both `CellToNodeRelation m_cell_node_rel` and `CellToNodeConnectivity* m_cell_to_node` ?

Reply:
I'll take a look at ripping that out -- it was easier to remove the ConnectivityArrays for face-based connectivity since we always own the memory for that.